tauri-app/src/lib/services/generateRainlangStrings.ts

```
@@ -0,0 +1,46 @@
+ */
+export async function generateRainlangStrings(
+  dotrainText: string,
+  settingsStrings: string[],
+  scenarios?: Record<string, ScenarioCfg>,
+): Promise<Map<ScenarioCfg, string> | undefined> {
+  try {
+    if (isEmpty(scenarios)) return undefined;
+
+    const composedRainlangForScenarios: Map<ScenarioCfg, string> = new Map();
+
+    for (const scenario of Object.values(scenarios!)) {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)